terraform/demo/main.tf

```diff
@@ -55,3 +55,15 @@ module "egress-space" {
     "steven.reilly@gsa.gov"
   ]
 }
+
+module "ses_email" {
+  source = "../shared/ses"
+
+  cf_org_name         = local.cf_org_name
+  cf_space_name       = local.cf_space_name
+  name                = "${local.app_name}-ses-${local.env}"
+  recursive_delete    = local.recursive_delete
+  aws_region          = "us-gov-west-1"
+  email_domain        = "sandbox.10x.gsa.gov"
```

Comment:
Should this be `notify.sandbox.10x.gsa.gov`?

Reply:
Yes, it should be, for now